neonvm/controllers/virtualmachine_controller.go

```diff
@@ -412,31 +415,36 @@ func (r *VirtualMachineReconciler) doReconcile(ctx context.Context, virtualmachi
 					return err
 				}
 				// compare guest spec and count of plugged
-				if *virtualmachine.Spec.Guest.CPUs.Use > int32(len(cpusPlugged)) {
+				if int32(virtualmachine.Spec.Guest.CPUs.Use.Value()) > int32(len(cpusPlugged)) {
 					// going to plug one CPU
```

Comment:
what is the type of `.Value()`?  type casting both to `int32` seems weird and if I were writing this I'd cast them both to plain `int`

Reply:
`Value()` is int64, and it rounds up